src/odemis/driver/technolution.py

```diff
@@ -541,22 +644,22 @@ def _setScanDelay(self, scanDelay):
                          "%s" % (scanDelay[0], self.parent._mppc.acqDelay.value))
             return self.scanDelay.value
 
-    def _setResolution(self, field_resolution):
+    def _setResolution(self, resolution):
         """
-        Sets the resolution so that the cell size (in both x and y direction) is a multiple of 4.
+        Sets the resolution so that the effective cell size (in both x and y direction) is a multiple of 4.
```

Comment:
I tried updating but I'm not sure if this is what you wanted.

Reply:
How about this:

`Sets the resolution of a single field image. The resolution of a single field image is the product of the number of cell images (detector shape) multiplied with the size of a cell image (effective cell size). Method sets the resolution of a single field image, so that the effective cell size (in x and y direction respectively) is a multiple of 4.`